actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -573,15 +573,15 @@ def current_page?(options = nil, check_parameters: false, **options_as_kwargs)
         end
       end
 
-      # Creates an SMS anchor link tag to the specified +phone_number+, which is
-      # also used as the name of the link unless +name+ is specified. Additional
-      # HTML attributes for the link can be passed in +html_options+.
+      # Creates an SMS anchor link tag to the specified +phone_number+. When the
+      # link is clicked, the default SMS messaging app is opened ready to send a
+      # message to the linked phone number.  If the +body+ option is specified,
```

Comment:
Nit picky but there's an extra space here.

Reply:
Good catch, thank you! :raised_hands: